### DISCUSSION TITLE
Zigbee 3.0 USB Dongle plus - don't work

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#12389] [General]
Waiting to understand if my problem [#12360](https://github.com/Koenkk/zigbee2mqtt/issues/12360) depends on the zigbee2mqtt version I bought one Sonoff dongle to make some test.
I removed the Conbee II, verified the location of the new adapter and setting the correct port in configuration.yaml and ......
zigbee2mqtt don't start and I have this error ...

`Error: network commissioning timed out - most likely network with the same panId or extendedPanId already exists nearby
`

How can I fix the issue? or how can I delete or change the old panId?
there must be the possibility to change the adapter without having to reinstall totally .....

Any help appreciated  ......

**Comment by Fillip38** [comment]
I have the same problem. I have testet two Sonoff Zigbee Sticks

Using '/app/data' as data directory
Zigbee2MQTT:info  2022-05-05 23:18:10: Logging to console and directory: '/app/data/log/2022-05-05.23-18-09' filename: log.txt
Zigbee2MQTT:info  2022-05-05 23:18:10: Starting Zigbee2MQTT version 1.25.1 (commit #3f6a137)
Zigbee2MQTT:info  2022-05-05 23:18:10: Starting zigbee-herdsman (0.14.27)
Zigbee2MQTT:error 2022-05-05 23:19:16: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-05-05 23:19:16: Failed to start zigbee
Zigbee2MQTT:error 2022-05-05 23:19:16: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-05-05 23:19:16: Exiting...
Zigbee2MQTT:error 2022-05-05 23:19:16: Error: network commissioning timed out - most likely network with the same panId or extendedPanId already exists nearby
    at ZnpAdapterManager.beginCommissioning (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:347:23)
    at ZnpAdapterManager.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:92:17)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:100:27)
    at start (/app/index.js:110:5)

**Reply by enrico-cc** [reply]
Similar issue for me:

openhabian@openhabian:/opt/zigbee2mqtt $ npm start

> zigbee2mqtt@1.25.1 start /opt/zigbee2mqtt
> node index.js

Zigbee2MQTT:info  2022-05-14 14:02:56: Logging to console and directory: '/var/log/zigbee2mqtt/2022-05-14.14-02-53al' filename: log.txt
Zigbee2MQTT:info  2022-05-14 14:02:56: Starting Zigbee2MQTT version 1.25.1 (commit #3f6a1378)
Zigbee2MQTT:info  2022-05-14 14:02:56: Starting zigbee-herdsman (0.14.27)
Zigbee2MQTT:error 2022-05-14 14:02:57: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-05-14 14:02:57: Failed to start zigbee
Zigbee2MQTT:error 2022-05-14 14:02:57: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-05-14 14:02:57: Exiting...
Zigbee2MQTT:error 2022-05-14 14:02:57: Error: Error while opening serialport 'Error: Error Resource temporarily unavailable Cannot lock port'
    at SerialPort.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)
    at SerialPort._error (/opt/zigbee2mqtt/node_modules/@serialport/stream/lib/index.js:198:14)
    at /opt/zigbee2mqtt/node_modules/@serialport/stream/lib/index.js:242:12
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.25.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zigbee2mqtt@1.25.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openhabian/.npm/_logs/2022-05-14T12_02_57_570Z-debug.log
